bigtable/client/rpc_backoff_policy.h

```diff
@@ -79,7 +80,8 @@ class ExponentialBackoffPolicy : public RPCBackoffPolicy {
       : current_delay_(std::chrono::duration_cast<std::chrono::microseconds>(
             initial_delay)),
         maximum_delay_(std::chrono::duration_cast<std::chrono::microseconds>(
-            maximum_delay)) {}
+            maximum_delay)),
+        generator_(std::random_device{}()) {}
```

Comment:
This initialization only adds 32-bits of entropy to the PRNG.  Please see:

https://github.com/GoogleCloudPlatform/google-cloud-cpp/pull/200/files#diff-4e6d93e1236d94e8f4047eba9182f34dR36

for an idea on how to properly initialize the generator.  Feel free to cut & paste the code to a function in bigtable::internal.

Reply:
Done.